ambry-api/src/main/java/com/github/ambry/account/Account.java

```diff
@@ -74,6 +76,19 @@
  * </p>
  */
 public class Account {
+  public static final QuotaResourceType QUOTA_RESOURCE_TYPE_DEFAULT_VALUE = QuotaResourceType.CONTAINER;
```

Comment:
nit: please put this line after line 106, that's where all the default values stay.

Reply:
done.